### PR TITLE
LPS-163132 [EVP] Create EVP Process Title

### PR DESCRIPTION
**Pull request by juliocezarjc**
[LPS-163132](https://issues.liferay.com/browse/LPS-163132)

### CODE REVIEW

**Comment by juliocezarjc** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/collection.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"description": "",
+	"name": "EVP"
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/collection.json:3]: Could you change the name to:
```suggestion
	"name": "EVP Fragments"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.evp-process .number-process{
+	font-size: 128px;
+	font-weight: normal;
+	line-height: 115px;
+	min-width: 0ch !important;
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css:5]: Is this line really necessary?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<div class="align-items-center d-flex evp-process">
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.html:1]: Could you rename the class to: 
```suggestion
<div class="align-items-center d-flex evp-process-title">
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.evp-process .number-process{
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css:1]: Could you rename the class to:
```suggestion
.evp-process-title .number-process{
```

Comment (JoMaAlves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css:1]: The same for the ones below

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+	font-weight: normal;
+	line-height: 23px;
+}
+
+.evp-process .process-title-container {
+	max-width: 400px;
+}
+
+.evp-process .process-icon{
+ width:40px
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.css:25]: Fix the indentation here.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+		<img alt="Placeholder" class="col process-icon" data-lfr-editable-id="image2" data-lfr-editable-type="image" src="[resources:file_template_icon.svg]" />
+
+		<img alt="Placeholder" class="col process-icon" data-lfr-editable-id="image3" data-lfr-editable-type="image" src="[resources:check_circle_icon.svg]" />
+
+		<img alt="Placeholder" class="col process-icon" data-lfr-editable-id="image4" data-lfr-editable-type="image" src="[resources:people_outline_icon.svg]" />
+
+		<img alt="Placeholder" class="col process-icon" data-lfr-editable-id="image5" data-lfr-editable-type="image" src="[resources:message_boards_icon.svg]" />
+		</div>
+	</div>
+</div>
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/evp-process-title/index.html:30]: Fix the indentation here.

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Pull request closed without merging**